### DISCUSSION TITLE
Cannot comment on tabby repo

### DISCUSSION THREAD

**Discussion by sino1641** [warp-tech/warpgate#1042] [General]
Hi and sorry to raise a new page here.
I cannot make any comments under tabby issue pr discussion, saying
`An owner of this repository has limited the ability to comment to users that have contributed to this repository in the past.`
May I ask if I have done anything violating the community or if it is just a mistake?
Much thanks.

**Comment by Eugeny** [comment]
I've disabled commenting for 24 hours due to a spam storm that's affecting large repos at Github right now
